<launch>
  <param name="device_id" value="4"/>
  <param name="calibration_path" value="$(find rpl)/config/camera.xml" />
  <node name="camera" pkg="rpl" type="camera.py" output="screen"/>
  <node name="calibration" pkg="rpl" type="calibration.py" output="screen"/>
  <node name="image_view" pkg="image_view" type="image_view" output="screen">
    <remap from="image" to="calib_image" />
  </node>
</launch>
